site/src/pages/components/menu.explainer.mdx

```diff
@@ -0,0 +1,439 @@
+---
+menu: Active Proposals
+name: Menu elements (Explainer)
+path: /components/menu.explainer
+layout: ../../layouts/ComponentLayout.astro
+---
+
+- Authors: [@domfarolino](https://github.com/domfarolino), [@dizhang168](https://github.com/dizhang168), [@josepharhar](https://github.com/josepharhar), [@mfreed7](https://github.com/mfreed7)
+
+{/* START doctoc generated TOC please keep comment here to allow auto update */}
+{/* DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE */}
+
+## Table of Contents
+
+- [Table of Contents](#table-of-contents)
+- [Introduction](#introduction)
+- [New `<menuitem>` elements](#new-menuitem-elements)
+- [Grouping `<menuitem>`s together](#grouping-menuitems-together)
+- [Question and Answers](#question-and-answers)
+  - [When to use Select vs Menu](#when-to-use-select-vs-menu)
+  - [Why are we adding `<menuitem>` and not re-using the `<command>` element?](#why-are-we-adding-menuitem-and-not-re-using-the-command-element)
+  - [Why are we adding `<menubar>` and `<menulist>` and not re-using the `<menu>` element?](#why-are-we-adding-menubar-and-menulist-and-not-re-using-the-menu-element)
+  - [Popover semantics for `<menulist>`](#popover-semantics-for-menulist)
+  - [Why are we using popover instead of openable for `<menulist>`?](#why-are-we-using-popover-instead-of-openable-for-menulist)
+  - [What should be the keyboard behavior?](#what-should-be-the-keyboard-behavior)
+  - [Should clicking/activating a menuitem close the menu by default?](#should-clickingactivating-a-menuitem-close-the-menu-by-default)
+  - [Should any of these elements be form-associated?](#should-any-of-these-elements-be-form-associated)
```

Comment:
You might limit the TOC to not include each question of the Q&A section. `doctoc` has a maxdepth option that can be used for this, FYI.